.github/workflows/build.yml

```
@@ -0,0 +1,20 @@
+
+name: Build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        name: Clone repo
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '14'
+        name: Set up Node
+      - run: npm ci 
+        name: Install dependencies
+      - run: npm run build
+        name: Build
```

```suggestion
      - name: Clone repo
        uses: actions/checkout@v2
      - name: Set up Node
        uses: actions/setup-node@v1
        with:
          node-version: '14'
      - name: Install dependencies
        run: npm ci 
      - name: Build
        run: npm run build
```

Name should come first so people know off the bat what each step does.